modules/installation/pages/pubcloud-requirements.adoc

```diff
@@ -33,6 +33,16 @@ Refer to the cloud provider documentation for detailed instructions.
 This prevents data loss if the instance is unexpectedly terminated.
 This section includes instructions for setting up an external virtual disk.
 
+
+[NOTE]
+====
+If you are attempting to bootstrap traditional clients, check that you can resolve the host name of the server while you are logged in to the client.
+You might need to add the FQDN of the server to [path]``/etc/hosts`` local resolution file on the client.
+Do this using the [command]``hostname -f`` command with the local IP address of the server.
```

Comment:
```suggestion
Check using the [command]``hostname -f`` command with the local IP address of the server.
```
Otherwise you could think that editing /etc/hosts will take place with the hostname command.